modules/ROOT/pages/single-sign-on-with-cas.adoc

```diff
@@ -33,84 +33,9 @@ As a consequence, configure the reverse proxy to include the `Host:` header when
 The method https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServletRequest.html#getRequestURL--[`getRequestURL ot HttpServletRequest`] is used internaly to retrieve the `server` and `port` of the HTTP request.
 ====
 
-==== example
```

Comment:
⚠️ are we sure we want to remove this whole block?

Reply:
it fix now